### PR TITLE
Lekse 120 create upload functionality

### PR DESCRIPTION
**Pull request by Gudzy**
- Add file upload functionality
- Add file download functionality 
- Add new question success page

### CODE REVIEW

File: src/views/NewQuestion/Sections/SectionForm.tsx [hunk 1]
```diff
@@ -24,30 +29,146 @@ const defaultOptions = {
...
+        files: results,
+      };
+      postQuestion(questionForm).then(() => {
+        history.push({ pathname: '/questions/new/success' });
+      });
+    });
+  };
+
+  function MyDropzone() {
+    const onDrop = useCallback(acceptedFiles => {
```
Comment (carrosa) [inline src/views/NewQuestion/Sections/SectionForm.tsx:126]: User has to be able to upload multiple files at once

File: src/views/NewQuestion/Sections/SectionForm.tsx [hunk 2]
```diff
@@ -24,30 +29,146 @@ const defaultOptions = {
...
+
+  const FileList = () => {
+    return (
+      <ul className="filelist">
+        {tempFiles.map((file, index) => {
+          const { name } = file;
+          return (
+            <li key={index}>
+              <span>
+                <a style={{ color: 'black', textDecoration: 'none' }} download>
```
Comment (carrosa) [inline src/views/NewQuestion/Sections/SectionForm.tsx]: spesiell grunn til at stylingen er her?

**Review by carrosa — APPROVED** [review]

File: src/views/NewQuestion/Sections/SectionForm.tsx (hunk 1 above)
Comment (Gudzy) [reply src/views/NewQuestion/Sections/SectionForm.tsx:126]: Godt observert! Fikset :)

File: src/views/NewQuestion/Sections/SectionForm.tsx (hunk 2 above)
Comment (Gudzy) [reply src/views/NewQuestion/Sections/SectionForm.tsx]: Nei, glemt å trekke den ut :) Fikset!

**Pull request merged**